File: Completed/Go/1672.go
func maximumWealth(accounts [][]int) int {
    maxWealth := 0
    for _, banks := range accounts {
        maxWealth = max(maxWealth, sumArr(banks))
    }
    return maxWealth
}

func sumArr (arr []int) int {
    sum := 0
    for _, v := range arr {
        sum += v
    }
    return sum
}

func max (a,b int) int {
    if a > b {
        return a
    }
    return b
}

File: Completed/python3/1672.py
import functools

class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        res = 0
        for banks in accounts:
            sumWealth = functools.reduce(lambda x,y : x+y, banks)
            res = max(res, sumWealth)
        return res
        
